DependencyInjection/Configuration.php

```diff
@@ -243,24 +243,16 @@ private function addAuthenticationPluiginNode()
                     ->then(function ($config) {
                         switch ($config['type']) {
                             case 'basic':
-                                if (empty($config['username']) || empty($config['password'])) {
-                                    throw new InvalidConfigurationException('Authentication "basic" requires both "username" and "password".');
-                                }
+                                Configuration::validateAuthenticationType(['username', 'password'], array_keys($config), 'basic');
```

Comment:
Stupid question: since it is a closure, why can't it be `$this->validateAuthenticationType`?